docs/contributor/04-30-ci-cd.md

```diff
@@ -1,51 +1,71 @@
+|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| [`post-istio-operator-build`](https://github.com/kyma-project/test-infra/blob/b21a8b638c841e985747645736ec78c7a38bc753/prow/jobs/kyma-project/istio/istio-manager.yaml#L355)                  | Builds Kyma Istio Operator's image and pushes it to the `prod` registry.                                                                          |
+| [`post-istio-module-build`](https://github.com/kyma-project/test-infra/blob/b21a8b638c841e985747645736ec78c7a38bc753/prow/jobs/kyma-project/istio/istio-manager.yaml#L208)                    | Builds ModuleTemplate for the Istio module and pushes it to the `prod` registry.                                                                  |
+| [`post-istio-integration`](https://github.com/kyma-project/test-infra/blob/b21a8b638c841e985747645736ec78c7a38bc753/prow/jobs/kyma-project/istio/istio-integration.yaml#L87)                  | Executes the main integration test suite for Istio Manager on a k3d cluster.                                                                      |
+| [`post-istio-integration-aws-specific`](https://github.com/kyma-project/test-infra/blob/b21a8b638c841e985747645736ec78c7a38bc753/prow/jobs/kyma-project/istio/istio-manager.yaml#L410)        | Executes the integration test suite specific for AWS Hyperscaler.                                                                                 |
+| [`main integration tests / Wait for image build`](https://github.com/kyma-project/istio/blob/26d8cfa69b91828fc34afd3b6aaf63c854d0cc54/.github/workflows/main-integration.yaml#L20)            | Waits for the operator's image build to finish so that image-dependent jobs can be executed.                                                      |
+| [`main integration tests / Istio upgrade integration tests`](https://github.com/kyma-project/istio/blob/26d8cfa69b91828fc34afd3b6aaf63c854d0cc54/.github/workflows/main-integration.yaml#L30) | Executes the upgrade integration test suite of Istio Module and verifies if the existing release can be successfully upgraded to the new version. |
+| [`main integration tests / Slack Notification`](https://github.com/kyma-project/istio/blob/26d8cfa69b91828fc34afd3b6aaf63c854d0cc54/.github/workflows/main-integration.yaml#L44)              | Sends a Slack notification to the team's channel if any pipelines fail on the main branch.                                                        |
+
+## Pipelines running on the PR during the release process
```

Comment:
We should have it similar to [api-gateway](https://github.com/kyma-project/api-gateway/blob/main/docs/contributor/04-30-ci-cd.md#pipelines-running-on-pull-requests-to-release-branches).
```suggestion
## Pipelines running on pull requests to release branches
```